components/config/definition.rst

```diff
@@ -568,7 +568,7 @@ tree with ``append()``::
     {
         $treeBuilder = new TreeBuilder('parameters');
 
-        $treeBuilder->getRootNode()
+        $node = $treeBuilder->getRootNode()
```

Comment:
won't just a return work here, instead of not neede variable?

Reply:
It would, how do I proceed?